docs/mkdocs/labsv2.md

```diff
@@ -0,0 +1,425 @@
+
+`load.yaml`
+```yaml
+---
+apiVersion: kmm.sigs.x-k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-a
+  namespace: labsv2 
+spec:
+  moduleLoader:
+    serviceAccountName: labsv2sa
+    container:
+      modprobe:
+        moduleName: kmm-ci-a
+      kernelMappings:
+        - regexp: '^.*'
+          containerImage: myrepo.xyz/kmm-labs:kmm-kmod-5.14.0-284.36.1.el9_2.x86_64-v1.0
+  imageRepoSecret: 
+    name: myrepo-pull-secret
+  selector:
+    node-role.kubernetes.io/worker: ""
+```
+
+## Replacing In-Tree Modules: Removing the existing Module before loading a new One
+
+We might need to remove an existing in-tree module before loading an out-of-tree one.
+Some use cases may be including the addition of more features, fixes, or patches to the out-of-tree module for the same driver, as well as encountering incompatibilities between the in-tree and out-of-tree modules.
+
+To achieve this we can remove in-tree modules just adding `inTreeModuleToRemove: <NameoftheModule>`. In our example we wil remove `joydev` module which is the standard in-tree driver included in the Linux kernel for joysticks or similar input devices:
+
+`remove_in-tree.yaml`
+```yaml
+---
+apiVersion: kmm.sigs.x-k8s.io/v1beta1
+kind: Module
+metadata:
+  name: kmm-ci-a
+  namespace: labsv2
+spec:
+  moduleLoader:
+    serviceAccountName: labsv2sa
+    container:
+      modprobe:
+        moduleName: kmm-ci-a
+      kernelMappings:
+        - regexp: '^.*'
+          containerImage: quay.io/ebelarte/kmm-labs:kmm-kmod-5.14.0-284.36.1.el9_2.x86_64-v1.0
+      inTreeModuleToRemove: joydev
+  imageRepoSecret: 
+    name: myrepo-pull-secret
+  selector:
+    node-role.kubernetes.io/worker: ""
+```
+
+We can confirm that `joydev` module is not loaded and our new module is loaded now just by listing the modules in one of our nodes and check that there is no output:
+```
+05:19:01 root@rhcs-sign-test labsv2-test → oc debug node/ebl-worker-1.edgeinfra.cloud 
+Temporary namespace openshift-debug-mr26h is created for debugging node...
+Starting pod/ebl-worker-1edgeinfracloud-debug ...
+To use host binaries, run `chroot /host`
+
+Pod IP: 192.168.122.101
+If you don't see a command prompt, try pressing enter.
+sh-4.4# 
+sh-4.4# chroot /host
+sh-5.1# lsmod | grep joydev
+sh-5.1# lsmod | grep kmm   
+kmm_ci_a               16384  0
+sh-5.1# 
+```
+
+## Device Plugins
+
+
+Device plugins play a crucial role in allocating and reporting specific hardware devices to the cluster. In KMM, you can load device plugin images as part of the Module object.
+
+For this example, we will utilize this plugin called [simple-device-plugin](https://github.com/yevgeny-shnaidman/simple-device-plugin) to simulate device plugins. This plugin provides a way to emulate device plugins and test their functionality within the cluster.
+
+It just loads some dummy devices settings in nodes as if they were reported by real hardware and then used by the moduleLoader.
```

Comment:
this sentence is not clear to me: what component is loading dummy device settings on the node?